here-naksha-lib-core/src/main/java/com/here/naksha/lib/core/models/storage/ReadFeatures.java

```diff
@@ -112,7 +112,7 @@ public void setCollections(@NotNull List<@NotNull String> collections) {
    * @param spatialOp The operation that should be applied to filter features by their geometry.
    * @return The previously assigned operation.
    */
-  public @Nullable SOp setPropertyOp(@Nullable SOp spatialOp) {
+  public @Nullable SOp setSpatialOp(@Nullable SOp spatialOp) {
```

Comment:
@xeus2001 / @cyberhead-pl - FYI - I changed name of the setters assuming it was a miss.